### DISCUSSION TITLE
Fast Refresh, CSS Modules and next/image

### DISCUSSION THREAD

**Discussion by eduardpauliuc** [vercel/next.js#60634] [Help]
### Summary
Editing css modules in app with `next/image` components causes full reload. 

### NextJS version
14.0.4

### Problem
While building an app, I ran into the following warning, for any changes in any `*.module.css` file (used in the current component tree):
``` ⚠ Fast Refresh had to perform a full reload. Read more: https://nextjs.org/docs/messages/fast-refresh-reload```

### Reproducing in create-next-app
I created a new app with `npx create-next-app@latest` (tried with node versions 18.19 and 20.10, got the same behaviour). When editing `page.module.css`, the warning is displayed. If I remove all `Image` components from `next/image`, the warnings are gone. 

App created with:
✔ Would you like to use TypeScript? … Yes
✔ Would you like to use ESLint? … Yes
✔ Would you like to use Tailwind CSS? … No
✔ Would you like to use `src/` directory? … Yes
✔ Would you like to use App Router? (recommended) … Yes
✔ Would you like to customize the default import alias (@/*)? … No

Other mentions: 
* jsx changes do not trigger full reload. 
* tried with both css / scss modules
* tried to remove `.next` and modules
* no components are lowercase (which I have read to be a problem)
* no arrow function components
* global css files does not trigger renders
* tried to use `unoptimized`

I have not seen a similar discussion/issue regarding `next/image`. 

### Additional information

```js
# Template package.json:
"dependencies": {
    "react": "^18",
    "react-dom": "^18",
    "next": "14.0.4"
  },
  "devDependencies": {
    "typescript": "^5",
    "@types/node": "^20",
    "@types/react": "^18",
    "@types/react-dom": "^18",
    "eslint": "^8",
    "eslint-config-next": "14.0.4"
  }
```

**Comment by Enkratia** [comment]
@eduardpauliuc hello, did you find any solution for this?
